src/components/login/Login.js

```diff
@@ -48,75 +51,76 @@ function Login(props) {
 
   return (
     <>
-      <When condition={!auth.isLoggedIn && !props.showSignup}>
-        <FormControl>
-          <Box sx={{ display: 'flex', flexWrap: 'nowrap' }}>
-            <div>
-              <Box sx={{ display: 'flex', alignItems: 'flex-end' }}>
-                <FormControl id="usernameLogin" sx={{ m: 1, width: '25ch' }} variant="outlined">
-                  <InputLabel htmlFor="outlined-adornment-username">Username</InputLabel>
-                  <OutlinedInput
-                    id="outlined-adornment-username"
-                    type={values ? 'text' : 'username'}
-                    value={values.username}
-                    onChange={handleChange('username')}
-                    endAdornment={
-                      <InputAdornment position="end">
-                        <AccountCircle sx={{ color: 'action.active', mr: -0.5, my: 0 }} />
-                      </InputAdornment>
-                    }
-                    label="Password"
-                  />
-                </FormControl>
+      <When condition={!auth.isLoggedIn && !props.
+        showSignup}>
+        <div className={theme.mode}>
```

Comment:
Looking forward to seeing how this looks! Implementing theme onto our components as well was something I thought of last night.